dash/include/dash/memory/GlobHeapMem.h

```diff
@@ -429,6 +429,23 @@ class GlobHeapMem
     return unit_local_size;
   }
 
```

Comment:
It would be interesting to see whether this ends up to be a `div` in the object code. I'd expect the compiler to optimize it since `sizeof(element_type)` is a compile-time constant.